themes/openy_themes/openy_rose/templates/page/page--node--landing-page--has-camp.html.twig

```diff
@@ -0,0 +1,53 @@
+{% extends "page.html.twig" %}
+
+{% block page_logo %}
+  {% set camp_section_image = '' %}
+  {% if camp_section_logo %}
+    {% set camp_section_image = file_url(camp_section_logo) %}
+    {% if camp_section_image is not empty %}
+      <a href="{{ base }}" title="Home" rel="home">
+        <img src="{{ camp_section_image }}" alt="Home">
+      </a>
+    {% else %}
+      {# Conditions not met use parent block. #}
+      {{ parent() }}
+    {% endif %}
+  {% else %}
+    {# Conditions not met use parent block. #}
+    {{ parent() }}
+  {% endif %}
+{% endblock %}
+
+{% block page_logo_2 %}
+  {% set camp_section_image = '' %}
+  {% if camp_section_logo %}
+    {% set camp_section_image = file_url(camp_section_logo) %}
+    {% if camp_section_image is not empty %}
+      <a href="/" title="Home" rel="home">
```

Comment:
possible issue with drupal installed into folder. Is it possible to pass ```base_path``` here? Or just check on QA level on a build